sample-app/src/main/java/com/splunk/android/sample/SecondFragment.java

```diff
@@ -93,9 +94,12 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
                 span.end();
             }
         });
-        binding.buttonToWebview.setOnClickListener(v ->
-                NavHostFragment.findNavController(SecondFragment.this)
-                        .navigate(R.id.action_SecondFragment_to_webViewFragment));
+        binding.buttonToWebview.setOnClickListener(v -> {
+            SplunkRum.getInstance().addRumEvent("this span will be ignored", Attributes.empty());
```

Comment:
doesn't this have to just be "ignored" to match the span filter defined in the config? Or should we update the span filter to have a `contains("ignored")`?

Reply:
🙈